flash_update/utils.py

```diff
@@ -58,7 +58,7 @@ def get_email_context(instance):
         ).only('id', 'file')
     }
     for resource in resources:
-        resource['flash_file'] = documents_map[resource['document']]
+        resource['flash_file'] = documents_map[resource['document']] if resource['document'] else None
```

Comment:
This should be fine.

```suggestion
        resource['flash_file'] = documents_map.get(resource['document'])
```